efabless_tool.py

```diff
@@ -181,6 +182,16 @@ def list_projects(projects, fields):
         logging.info(log)
 
 
+def list_by_ip(projects,ip):
+    ip=re.sub("-","",ip)
+    for project in projects:
+        if re.search(ip,project["summary"],re.IGNORECASE):
+            if project["giturl"] != "n/a":
+                logging.info("%-5s %-35s %-80s" % (project["id"], project["owner"], project["giturl"]))
```

Comment:
use fstrings instead of % strings

Reply:
see the way that the list function works. And use the widths specified in the dict at the top of the program instead of hardcoding here.

Reply:
OK. let me do that.